app/src/main/java/com/codesoom/assignment/dto/SessionResponseData.java

```diff
@@ -0,0 +1,12 @@
+package com.codesoom.assignment.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
```

Comment:
`@Builder`를 클래스 시그니처 위에 붙여주지 않는 것이 좋습니다. (다행히 이 클래스에는 멤버가 하나 뿐이지만요) 그 이유에 대해 조사해 보세요.

Reply:
공식문서에 따르면 다음과 같이 말하고 있습니다.

> Finally, applying @Builder to a class is as if you added @AllArgsConstructor(access = AccessLevel.PACKAGE) to the class and applied the @Builder annotation to this all-args-constructor. This only works if you haven't written any explicit constructors yourself. If you do have an explicit constructor, put the @Builder annotation on the constructor instead of on the class. Note that if you put both `@Value` and `@Builder` on a class, the package-private constructor that `@Builder` wants to generate 'wins' and suppresses the constructor that `@Value` wants to make.

`@Builder`를 클래스에 적용하는 것은 `@AllArgsConstructor(access = AccessLevel.PACKAGE)`를 클래스에 적용하고 `@Builder` annotation을 all-args-constructor에 붙인 것과 같다. 이것은 명시적인 생성자를 선언하지 않았을 경우에만 동작한다. 만약 명시적으로 선언한 생성자가 있을 경우, `@Builder` annotation을 class가 아닌 생성자에 붙여라. `@Value`와 `@Builder` 두 가지 모두를 클래스에 달아 줄 경우 `@Builder`가 'wins'를 생성하려고 하는 package-private 생성자(JAVA에서 접근자 키워드가 없이 선언되는 생성자)가 생성되고 `@Value`가 생성하려는 생성자가 표시되지 않는것에 주의하라.

클래스 시그니처 위에 붙이는 것은 `@AllArgsConstructor(access = AccessLevel.PACKAGE)`를 클래스에 적용하는 것과 같으므로 `@AllArgsConstructor` 를 사용하면 발생할 수 있는 문제점을 내포하고 있습니다.

따라서 직접 만든 생성자 위에 붙여서 사용하는걸 권장하는 것 같습니다.